ConcernsCaseWork/ConcernsCaseWork.Integration.Tests/appsettings.tests.json

```diff
@@ -26,7 +26,7 @@
 	"api_key": "secret"
   },
   "ConnectionStrings": {
-	"DefaultConnection": "Server=localhost;Database=sip;Integrated Security=true;"
```

Comment:
is this change expected?

Reply:
Yes

Reply:
It is so that the integration tests aren't running against the same database as the api